lib/rouge/lexers/yara.rb

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class YARA < RegexLexer
+      title "YARA"
+      desc ""
```

Comment:
Could you add a description here? You can see examples in the Ruby lexer

https://github.com/rouge-ruby/rouge/blob/96efe28341c087fe4d4e0f2360043b7a632ea562/lib/rouge/lexers/ruby.rb#L8

and in the JavaScript lexer

https://github.com/rouge-ruby/rouge/blob/96efe28341c087fe4d4e0f2360043b7a632ea562/lib/rouge/lexers/javascript.rb#L14